lib/Logger.js

```diff
@@ -0,0 +1,86 @@
+let LogLevel = 0;
+
+const LogLevels = {
+    trace: -2,
+    debug: -1,
+    info: 0,
+    warn: 1,
+    error: 2
+};
```

Comment:
You can wrap this in Object.freeze since there are no plans for extensions.

The keys should be quoted since they are used as strings elsewhere (in minification non-quoted keys may get renamed).